jest.config-e2e.js

```diff
@@ -19,13 +19,14 @@
  * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
  * @link      https://github.com/serlo-org/serlo.org for the canonical source repository
  */
-const { mergeDeepRight } = require('ramda')
-
 const puppeteerPreset = require('jest-puppeteer/jest-preset')
 const tsPreset = require('ts-jest/jest-preset')
 
 module.exports = {
-  ...mergeDeepRight(puppeteerPreset, tsPreset),
+  ...puppeteerPreset,
+  ...tsPreset,
```

Comment:
ping @kulla

Reply:
Yes, this was intentional. In the previous code the setting `setupFilesAfterEnv` of `puppeteerPreset` was overwritten and thus we couldn't use https://github.com/smooth-code/jest-puppeteer/blob/master/packages/expect-puppeteer/README.md#api

`mergeDeepRight` does not concat list properties but uses the list of the second dictionary. Therefore I have deleted this function so that developers are aware that they have to look at the dictionaries and how the configs needed to be merged. (In the case of `puppeteerPreset` and `tsPreset` they have different properties).

A better solution and a Follow-Up is a mergeConfig function based on `R.mergeWith` which concats list and can be used here.

Reply:
Okay, I have added a `mergeConfig` doing what is needed.